deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml

```diff
@@ -30,6 +30,51 @@ spec:
         metadata:
           type: object
         spec:
+          properties:
+            backups:
+              description: Backups is an array of configs that will be used to create
+                CronJob resource instances
+              items:
+                properties:
+                  backendSecretName:
+                    description: BackendSecretName is the name of a secret containing
+                      storage backend details, such as "AWS_S3_BUCKET_NAME", "AWS_ACCESS_KEY_ID",
+                      and "AWS_SECRET_ACCESS_KEY"
+                    type: string
+                  backendSecretNamespace:
+                    description: BackendSecretNamespace is the name of the namespace
+                      that the secret referenced in BackendSecretName resides in
+                    type: string
+                  encryptionKeySecretName:
+                    description: EncryptionKeySecretName is the name of a secret containing
+                      PGP/GPG details, including "GPG_PUBLIC_KEY", "GPG_TRUST_MODEL",
+                      and "GPG_RECIPIENT"
+                    type: string
+                  encryptionKeySecretNamespace:
+                    description: EncryptionKeySecretNamespace is the name of the namespace
+                      that the secret referenced in EncryptionKeySecretName resides
+                      in
+                    type: string
+                  labels:
+                    additionalProperties:
+                      type: string
+                    description: Labels to be added to the resulting CronJob
+                    type: object
+                  name:
+                    description: Name is the name that will be given to the resulting
+                      CronJob
+                    type: string
+                  schedule:
+                    description: Schedule is the schedule that the job will be run
+                      at, in cron format
+                    type: string
+                required:
+                - name
```

Comment:
Required should be just the AWS secret name and/or the details. All info can be generated and obtained dynamically. 
In MSS Oper we also have as standard the mandatory_specs.go which will set default values when nothing is informed. In this case see here: https://github.com/aerogear/mobile-security-service-operator/blob/80cc86edcbb37fabf63c6e1c561be574510d9d49/pkg/controller/mobilesecurityservicebackup/mandatory_specs.go

Reply:
I've removed the labels, but I believe the name and schedule should always be specified.